web-client/.env.EXAMPLE

```diff
@@ -1,12 +0,0 @@
-REACT_APP_API_BASE_URL=
-REACT_APP_FACEBOOK_APP_ID=
-REACT_APP_FIREBASE_API_KEY=AIzaSyDU5FZEGBwU9lv5RxSQ4ObPPaqhX_PBnYk
-REACT_APP_FIREBASE_AUTH_DOMAIN=reach4help-dev.firebaseapp.com
-REACT_APP_FIREBASE_DATABASE_URL=https://reach4help-dev.firebaseio.com
-REACT_APP_FIREBASE_PROJECT_ID=reach4help-dev
-REACT_APP_FIREBASE_STORAGE_BUCKET=reach4help-dev.appspot.com
-REACT_APP_FIREBASE_MESSAGING_SENDER_ID=664078449280
-REACT_APP_FIREBASE_APP_ID=1:664078449280:web:115f33485dfc48e35f4465
-REACT_APP_FIREBASE_MEASUREMENT_ID=G-DKH4PLZ62L
-REACT_APP_GMAPS_API_KEY=AIzaSyD-BZLD8hJfy8AXQsUf9LC183t0PSj12XE
-TEMP_ALGOLIA_SEARCH_KEY=be6dcf00f35efe86aecaf1887912b918
```

Comment:
Probably best not to delete this file for new devs.

Reply:
Ah, must have been a mistake... Fixing it now